src/main/resources/project/jython/stop_cluster.jython

```diff
@@ -38,15 +39,46 @@ clusterName = r'''
 [% cluster_name %]
 '''.strip()
 
+timeout = r'''
+[% timeout %]
```

Comment:
I know, still I don't get why it has a different name. forceShutdown for example has the same name as a parameter.